sbin/common/sbom.sh

```diff
@@ -7,7 +7,15 @@ createSBOMFile() {
   "${javaHome}"/bin/java -cp "${classpath}" temurin.sbom.TemurinGenSBOM --createNewSBOM --jsonFile "${jsonFile}"
 }
 
-# Set basic SBMO metadata with timestamp, authors, manufacture to ${sbomJson}
+# sign SBOM json file: sbomJson
+signSBOMFile() {
+  local javaHome="${1}"
+  local classpath="${2}"
+  local jsonFile="${3}"
```

Comment:
Either it will be a List of sbom json files.
or we need to call signSBOMFile many times when loop over these json files.